tests/resolvers/Mutation/removeUserFromUserFamily.spec.ts

```diff
@@ -54,12 +54,17 @@ describe("resolver -> Mutation -> removerUserFromUserFamily", () => {
       .spyOn(requestContext, "translate")
       .mockImplementation((message) => `Translated ${message}`);
     try {
+      if (testUserFamily == null) {
+        throw Error("Error in test");
+      }
```

Comment:
We don't need this. We can always assume that our helpers would return a valid `testUserFamily`

Reply:
Then I have to use `es-lint ignore`. I was trying to get rid of these statements but unfortunately linter doesn't allow! I saw in a thread that someone is working on fixing this issue. But till that, ignore statements are the only way!